### ISSUE TITLE
Marker md5 files contain absolute path, should just be file name

### ISSUE THREAD

**Issue by chasemc** [KwanLab/Autometa#149]
https://github.com/KwanLab/Autometa/blob/50f7a60c329acfa9e705b4cf343dfdcc21ade0ec/autometa/databases/markers/archaea.single_copy.hmm.md5#L1

https://github.com/KwanLab/Autometa/blob/50f7a60c329acfa9e705b4cf343dfdcc21ade0ec/autometa/databases/markers/bacteria.single_copy.hmm.md5#L1

**Comment by WiscEvan** [comment]
**TL; DR: In either case, the filename or the relative path to the file in the repo will not change the hash. Maybe this is a preference thing? I'm not sure this is going to change behavior at all.** 

_________

These are relative paths, not absolute paths. These are relative to the root directory of the repository at the location where these files are hosted.

This could be altered if you would like it to be more explicit

```bash
# Note now we are specifying the current Autometa root directory to begin the relative path
./autometa/databases/markers/bacteria.single_copy.hmm
# The above is implied from below
autometa/databases/markers/bacteria.single_copy.hmm 
```

### Example Absolute paths

Below would be considered absolute if a root directory contained a directory called `/autometa`.

```bash
# This is an invalid filepath for this repository (Missing Autometa root dir)
/autometa/databases/markers/bacteria.single_copy.hmm
# This is *not* using the user's $HOME directory and therefore would require root level permissions/privileges.
/Autometa/autometa/databases/markers/bacteria.single_copy.hmm
```

The absolute path could also look something like the following:

```bash
~/Autometa/autometa/databases/markers/bacteria.single_copy.hmm
```

tilde has been aliased for `$HOME`. e.g.

```bash
$HOME/Autometa/autometa/databases/markers/bacteria.single_copy.hmm
```
Since we do not know whether the user has root directory permissions/privileges and we do not know the user's `$HOME` directory, we are opting to point to the path relative to the repository file structure.

**Comment by WiscEvan** [comment]
# Submit this as a PR to fix

## Fix archaea markers md5 file

### Issue
> https://github.com/KwanLab/Autometa/blob/50f7a60c329acfa9e705b4cf343dfdcc21ade0ec/autometa/databases/markers/archaea.single_copy.hmm.md5#L1
> 

### Fix

Updated contents of `Autometa/autometa/databases/markers/archaea.single_copy.hmm.md5`

```bash
d80561bc637bfe65ffb6a975bfa07b8e  archaea.single_copy.hmm 
```

## Fix bacteria markers md5 file

### Issue
> https://github.com/KwanLab/Autometa/blob/50f7a60c329acfa9e705b4cf343dfdcc21ade0ec/autometa/databases/markers/bacteria.single_copy.hmm.md5#L1

### Fix

Updated contents of `Autometa/autometa/databases/markers/bacteria.single_copy.hmm.md5`

```bash
 3a32f37b3e269b6cc4fc0e440984c3d8  bacteria.single_copy.hmm 
```

**Comment by chasemc** [comment]
> **TL; DR: In either case, the filename or the relative path to the file in the repo will not change the hash. Maybe this is a preference thing? I'm not sure this is going to change behavior at all.**

`md5sum -c bacteria.single_copy.hmm.md5` will fail unless run from the path that the relative path is true

**Comment by chasemc** [comment]
Completed, as seen here:
https://github.com/KwanLab/Autometa/blob/36d52b18425d765a804a09f020bc2bda8374094c/autometa/databases/markers/archaea.single_copy.hmm.md5
https://github.com/KwanLab/Autometa/blob/36d52b18425d765a804a09f020bc2bda8374094c/autometa/databases/markers/bacteria.single_copy.hmm.md5

**Comment by ReneKat** [comment]
Hello there KwanLab!
The most recent release doesn't have the same directories as the original SCGs. 
 Autometa/autometa/databases/markers/archaea.single_copy.hmm.md5 no longer exists. 
The single-copy-markers directory doesn't have any md5 files for md5sum to check. Therefore, when I run autometa it fails due to md5sum. 
Best,
Rene

**Comment by chasemc** [comment]
Hello, 
I think think this is a separate issue, would you mind opening a new bug report that includes some information like what version of Auotometa you're currently using and which command is failing? (You can do that here: https://github.com/KwanLab/Autometa/issues/new?assignees=&labels=&template=bug_report.md)

The reason I ask is because
`single-copy-markers` is  directory on the main branch
and
`Autometa/autometa/databases/markers/archaea.single_copy.hmm.md5` is a path in the `dev` (currently being developed) branch and is structured very differently than the main branch

**Comment by ReneKat** [comment]
Hi Chase! Thanks for your quick response. 
I was getting ahead of myself and hadn't even made the databases directory yet by running `make_taxonomy_table.py`! Please disregard my comment above. 
Best,
Rene
